Traceback (most recent call last):
  File "timer_script.py", line 1, in <module>
    import schedule
ImportError: No module named schedule
Traceback (most recent call last):
  File "timer_script.py", line 1, in <module>
    import schedule
ImportError: No module named schedule
Traceback (most recent call last):
  File "timer_script.py", line 1, in <module>
    import schedule
ImportError: No module named schedule
Traceback (most recent call last):
  File "timer_script.py", line 1, in <module>
    import schedule
ImportError: No module named schedule
Traceback (most recent call last):
  File "timer_script.py", line 1, in <module>
    import schedule
ImportError: No module named schedule
Traceback (most recent call last):
  File "timer_script.py", line 1, in <module>
    import schedule
ImportError: No module named schedule
/Users/davinci/travAil/.venv/bin/python: can't open file '/Users/davinci/travAil/scripts/scripts/main.py': [Errno 2] No such file or directory
/Users/davinci/travAil/.venv/bin/python: can't open file '/Users/davinci/travAil/scripts/scripts/main.py': [Errno 2] No such file or directory
